python/lsst/validate/drp/validate.py

```diff
@@ -319,12 +321,11 @@ def add_measurement(measurement):
         afxName = 'AF{0:d}'.format(x)
         adxName = 'AD{0:d}'.format(x)
 
-
         amx = measureAMx(metrics['validate_drp.'+amxName], matchedDataset, D*u.arcmin)
         add_measurement(amx)
 
-        afx_spec_set = specs.subset(required_meta={'instrument':'HSC'}, spec_tags=[afxName,])
-        adx_spec_set = specs.subset(required_meta={'instrument':'HSC'}, spec_tags=[adxName,])
+        afx_spec_set = specs.subset(required_meta={'instrument': 'HSC'}, spec_tags=[afxName, ])
+        adx_spec_set = specs.subset(required_meta={'instrument': 'HSC'}, spec_tags=[adxName, ])
```

Comment:
Do you really want HSC hard-coded here? The value is `instrument` below. I realize this is pre-existing code.

Reply:
I'm not going to modify the existing code other than to flake8 clean it: @SimonKrughoff or @wmwv would have to decide what appropriate defaults are here.